src/components/PageHeaderCompact/PageHeaderCompact.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+
+import Author, { AuthorProps } from 'Author/Author';
+import { ImageElement } from 'Image';
+import RichText from 'RichText';
+import TagList, { TagProps } from 'TagList/TagList';
+import SocialShare from 'SocialShare';
+import FormattedDate from 'FormattedDate';
+
+interface AuthorPropsExtend extends AuthorProps {
```

Comment:
type is preferable to interface as interface can easily inadvertently acquire props from other components, in which case you can change this to 

```type AuthorPropsExtend = AuthorProps &&  {```

Example in use for the Card component
https://github.com/wellcometrust/corporate-components/blob/e4812b145f31b2db9fb84db0460778c6beaddf49/src/components/Card/Card.tsx#L18

Reply:
ah amazing, I thought had to use an interface in order to extend. will update this